monkey/monkey_island/cc/services/config_schema/ransomware.py

```diff
@@ -6,17 +6,14 @@
 RANSOMWARE = {
     "title": "Ransomware",
     "type": "object",
-    "description": "This page allows you to configure the Infection Monkey to execute a ransomware "
-    "simulation. The Infection Monkey is capable of simulating a ransomware attack on your network "
-    "using a set of configurable behaviors. A number of precautions have been taken to ensure that "
-    "this ransomware simulation is safe for production environments.\n\nFor more information about "
-    "configuring the ransomware simulation, see "
-    '<a href="https://guardicore.com/infectionmonkey/docs/usage/use-cases/ransomware-simulation" '
-    'target="_blank"> the documentation</a>.',
     "properties": {
         "encryption": {
-            "title": "Encryption",
+            "title": "Simulation",
             "type": "object",
+            "description": "To simulate ransomware encryption, create a directory and put some "
+            "files there to be encrypted. Do this on each machine on which you want to run the "
+            "ransomware encryption simulation.\n\nProvide the path to the directory that was "
```

Comment:
@mssalvatore  can we do "Do this on each machine you want to run the ransomware encryption simulation." to make this statement shorter/more straightforward?

Reply:
```suggestion
            "description": "To simulate ransomware encryption, you'll need to provide Infection "
            "Monkey with files that it can safely encrypt. On each machine where you would like to "
            "run a ransomware simulation, create a directory and put some files in it."
            "\n\nProvide the path to the directory that was created on each machine."
```